### DISCUSSION TITLE
How to deal with incomplete chloroplast after adjusting many parameters ?

### DISCUSSION THREAD

**Discussion by curtainDian** [Kinggerm/GetOrganelle#202] [Q&A]
Mr. Kinggerm, I adjusted many parameters( decrease -w (from 60-95); max-reads inf; reduce-reads-for-coverage inf ; change reference chloroplast ), but there are still some samples which I can't get chloroplast from. From the log, "incomplete edge" and "mutiple isolated embplant_pt components detected ", both of them above appear frequently. How can I deal with the remaining incomplete results? Could you please offer more suggestions ? Thanks a lot ! one of the log files of failed assembly is below. 
[get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/9757315/get_org.log.txt)

**Comment by jjiami** [comment]
请问问题解决了吗？我用重测序数据组装叶绿体基因组，结果是成环的，但和参考比总是缺少1500bp左右，导致IRa区的最后一个基因rps19总是残缺的，但我用同样的方法组装另一个很近的物种就没有出现这种问题。您有什么想法吗？

**Reply by Kinggerm** [reply]
Please check the IR boundary movement (expansion/contraction) in the literature.

**Comment by Kinggerm** [comment]
> Mr. Kinggerm, I adjusted many parameters( decrease -w (from 60-95); max-reads inf; reduce-reads-for-coverage inf ; change reference chloroplast ), but there are still some samples which I can't get chloroplast from. From the log, "incomplete edge" and "mutiple isolated embplant_pt components detected ", both of them above appear frequently. How can I deal with the remaining incomplete results? Could you please offer more suggestions ? Thanks a lot ! one of the log files of failed assembly is below. [get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/9757315/get_org.log.txt)

For incomplete results, all guidelines we had were listed [here](https://github.com/Kinggerm/GetOrganelle/wiki/FAQ#what-should-i-do-with-incomplete-resultbroken-assembly-graph). I don't think I have enough information to offer recommendations. However, you should always check the assembly graph before making any parameter adjustments.
There is no universal solution for all samples. A certain amount of samples could even contain gaps in the reads.
